test/rubygems/test_gem_remote_fetcher.rb

```diff
@@ -5,8 +5,7 @@
 begin
   require 'webrick/https'
 rescue LoadError => e
-  raise unless (e.respond_to?(:path) && e.path == 'openssl') ||
-               e.message =~ / -- openssl$/
+  raise unless e.path == 'openssl'
```

Comment:
Do you want me to also remove this change, since it's also unrelated to `rake`?

Reply:
No big deal